main.nf

```diff
@@ -94,6 +94,8 @@ anno_list = define_anno_list()
 annotate_tools = params.annotate_tools ? params.annotate_tools.split(',').collect{it.trim().toLowerCase().replaceAll('-', '')} : []
 if (!check_parameter_list(annotate_tools,anno_list)) exit 1, 'Unknown tool(s) to annotate, see --help for more information'
 
+if (!(params.aligner in ['bwa-mem', 'bwa-mem2'])) exit 1, 'Unknown aligner, see --help for more information'
```

Comment:
Should we keep both aligners or remove bwa-mem? Maybe depends on whether bwa-mem is further developed and not replaced with bwa-mem2

Reply:
I'd say let's keep both for now, but let's keep that question in mind.
https://github.com/nf-core/sarek/issues/337